src/client/components/back-button/back-button.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+  <header class="back-button__container">
+    <md-button class="md-raised" @click="$router.go(-1)">
```

Comment:
The router has a `.back()` method. I think that is more readable then `.go(-1)`

Reply:
changed and agreed. I just copied that over